clients/banking/src/components/TransferRecurringWizardDetails.tsx

```diff
@@ -74,7 +74,7 @@ const TransferRecurringWizardDetailsFixedAmount = ({
       validate: value => {
         const amount = Number(value);
 
-        if (Number.isNaN(amount) || value === "" || amount <= 0) {
```

Comment:
not the right type of recurring transfer, the issue in on target account balance, not fixed amount